02_LESSON/about.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>About</title>
+        <link rel="icon" href="pngwing.com.png" type="image/x-icon">
+        <meta name="description" content=" Hola Hedwig Van Braco, this page is about me">
```

Comment:
Hay que cerrar este tag así:

`        <meta name="description" content=" Hola Hedwig Van Braco, this page is about me" />